plugins/circleci/src/circleci-config.ts

```diff
@@ -123,6 +123,11 @@ const mergeWithConcatenatedArrays = (arg0: unknown, ...args: unknown[]) =>
   })
 
 const getBaseConfig = (): CircleCIState => {
+  const runsOnMultipleNodeVersions = getNodeVersions().length > 1
+  const setupMatrix = {
+    ...(runsOnMultipleNodeVersions ? { name: 'tool-kit/setup-<< matrix.executor >>' } : {}),
+    ...(runsOnMultipleNodeVersions ? matrixBoilerplate() : { executor: 'node' })
```

Comment:
**suggestion**: this could be simplified by moving the `name: 'tool-kit/setup-<< matrix.executor >>'` into `matrixBoilerplate`, then you'd only need one spread/ternary